package.json

```diff
@@ -41,33 +41,33 @@
   },
   "homepage": "https://github.com/silinternational/ui-components#readme",
   "peerDependencies": {
-    "material-components-web": "^10.0.0",
+    "material-components-web": "^13.0.0",
```

Comment:
Does this change mean that we will have to bump our major version when we release these changes (i.e. are we making a backwards-compatibility-breaking change to our library by doing this)?

Reply:
yes, If you click on this [commit](https://github.com/silinternational/ui-components/pull/140/commits/6e44b7644e28083646bcaaedc9dbfedaf0f978b4) You can see I included BREAKING CHANGES: ... which will trigger a breaking change to 6.0.0